<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDefaultParametersDialog</class>
 <widget class="QDialog" name="SetDefaultParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set default parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="algorithmLabel">
       <property name="text">
        <string>Weight algorithm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="algorithmComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="scoreLabel">
       <property name="text">
        <string>Score:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="scoreSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>85</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scoreValueLabel">
         <property name="text">
          <string>85%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
